test_infra/src/lib.rs

```diff
@@ -264,7 +264,7 @@ impl DiskConfig for UbuntuDiskConfig {
         fs::File::open(source_file_dir.join("user-data"))
             .unwrap()
             .read_to_string(&mut user_data_string)
-            .expect("Expected reading user-data file in to succeed");
+            .expect("Expected reading user-data file to succeed");
```

Comment:
Unlike the `in to` => `into`; here I don't think `in` adds anything, so I'm removing it...